src/Microsoft.Health.Fhir.Core/Models/ModelInfoProvider.cs

```
@@ -14,6 +14,10 @@ public static class ModelInfoProvider
     {
         private static IModelInfoProvider _modelInfoProvider;
 
+        public static FhirSpecification Version => _modelInfoProvider?.Version ?? FhirSpecification.Stu3;
```

nit: maybe throw exception rather than automatically default to STU3?

---

(perhaps call EnsureProvider() before returning the instance here and below.)